src/app/app.component.scss

```diff
@@ -22,6 +26,25 @@
   }
 }
 
+.fullscreen {
```

Comment:
Doubt that position and z-index are needed! Also sizes seems off!? And use correct nesting (look at the existing styles!)

BIG NOTE: Use rem NOT em! I have pointed this out multiple times!